apps/native-component-list/src/screens/DocumentPickerScreen.tsx

```diff
@@ -33,39 +36,68 @@ export default function DocumentPickerScreen() {
   };
 
   return (
-    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
-      <Button onPress={openPicker} title="Open document picker" />
-      <TitleSwitch
-        style={{ marginVertical: 10 }}
-        value={copyToCache}
-        setValue={setCopyToCache}
-        title="Copy to cache"
-      />
-      <TitleSwitch
-        style={{ marginVertical: 10 }}
-        value={multiple}
-        setValue={setMultiple}
-        title="Pick multiple"
-      />
+    <View
+      style={{
+        flex: 1,
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}>
+      <View
+        style={{ marginBottom: 20, marginTop: 20, paddingHorizontal: 20, gap: 5, minWidth: 300 }}>
+        <Button onPress={openPicker} title="Open document picker" buttonStyle={{ width: '100%' }} />
+        <TitleSwitch
+          style={{ marginVertical: 10 }}
+          value={copyToCache}
+          setValue={setCopyToCache}
+          title="Copy to cache"
+        />
+        <TitleSwitch
+          style={{ marginVertical: 10 }}
+          value={multiple}
+          setValue={setMultiple}
+          title="Pick multiple"
+        />
+        <TitleSwitch
+          style={{ marginVertical: 10 }}
+          value={base64}
+          setValue={setBase64}
+          title="Base64 (web only)"
+        />
+      </View>
+
       <FlatList
         data={pickerResult?.assets}
+        contentContainerStyle={{
+          padding: 20,
+          maxWidth: width - 40,
+          justifyContent: 'flex-start',
+        }}
         keyExtractor={(item, index) => `${index}-${item.uri}`}
         renderItem={({ item: document }) => {
           return (
-            <View>
+            <View style={{ marginBottom: 20, width: '100%', flex: 1 }}>
               {document.name!.match(/\.(png|jpg)$/gi) ? (
                 <Image
                   source={{ uri: document.uri }}
                   resizeMode="cover"
                   style={{ width: 100, height: 100 }}
                 />
               ) : null}
-              <Text>
+              <Text numberOfLines={1} ellipsizeMode="middle">
                 {document.name} ({document.size! / 1000} KB)
               </Text>
-              <Text>
-                URI: {document.uri} MimeType: {document.mimeType}
+              {document.base64 ? (
+                <Text numberOfLines={1} ellipsizeMode="middle">
+                  Base64: {document.base64.slice(0, 50)}...
+                </Text>
+              ) : null}
+              <Text numberOfLines={1} ellipsizeMode="middle">
+                URI: {document.uri}
+              </Text>
+              <Text numberOfLines={1} ellipsizeMode="middle">
+                MimeType: {document.mimeType}
               </Text>
+              <Text>Last Modified: {document.lastModified}</Text>
```

Comment:
Just truncated the preview, otherwise the layout would be completely broken on web. The console.log is still available with all the info